src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import duke.commands.Command;
+
+/**
+ * Main driver class for Duke
+ */
+public class Duke {
+    public static final String TERMINATION_INDICATION = "TERMINATE";
+    private TaskList tasks;
```

Comment:
I like how you differentiated singular vs plural variables in your variable naming!